src/main/java/com/google/sps/servlets/SpoonacularNutritionServlet.java

```diff
@@ -39,8 +44,13 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String title = request.getParameter("dishName");
-    title = title.replaceAll(" ", "+");
     Client client = ClientBuilder.newClient();
+    try {
+      title = URLEncoder.encode(title);
+    }
+    catch (Exception e) {
+      System.out.println(e);
+    }
```

Comment:
Maybe you can just let it throw exceptions, then no need to write the try-catch block